week1/assignment2/index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>💸용감자의 가계부💸</title>
+  </head>
+  <body>
+    <div id="wrap">
+      <!-- 제목 -->
+      <header>💸 용감자의 가계부 💸</header>
```

Comment:
header 는 제목, 로고, 검색 폼, 작성자 이름 등의 요소 를 포함하는 소개 및 탐색에 도움을 주는 콘텐츠를 담는 영역!이기 때문에 정확히는 이 안에서 
```suggestion
      <header><h1>💸 용감자의 가계부 💸</h1></header>
```
이렇게 써주는게 좋답니다! :)